backend/src/bin/jobs/mergeSuggestions.ts

```diff
@@ -0,0 +1,27 @@
+import TenantService from '../../services/tenantService'
+import { CrowdJob } from '../../types/jobTypes'
+import { sendNodeWorkerMessage } from '../../serverless/utils/nodeWorkerSQS'
+import { NodeWorkerMessageType } from '../../serverless/types/workerTypes'
+import { NodeWorkerMessageBase } from '../../types/mq/nodeWorkerMessageBase'
+import { timeout } from '../../utils/timing'
+
+const job: CrowdJob = {
+  name: 'Merge suggestions',
+  // every hour
+  cronTime: '0 * * * *',
+  onTrigger: async () => {
```

Comment:
We are triggering this once per hour?
For each tenant and after each tenant we wait for 2 seconds.
This means that at 1800 tenants one trigger will last for 1 hour and it will immediately start again.

I suggest to check `checkstuckIntegrationRuns.ts` -> `running` flag and implement something similar to prevent check starting if one is already in progress.

Reply:
For now, I will reduce the wait time to 1s, which will be fine, and we can add this later. I would like the feature to ship ASAP, because rn we don't have merge suggestions